libraries/SITL/SIM_Frame.cpp

```diff
@@ -322,10 +324,15 @@ float Frame::get_air_density(float alt_amsl) const
     return air_pressure / (ISA_GAS_CONSTANT * (C_TO_KELVIN(model.refTempC)));
 }
 
-#if USE_PICOJSON
+#if AP_JSONPARSER_ENABLED
 /*
   load frame specific parameters from a json file if available
  */
+
+#ifndef streq
+#define streq(a,b) !strcmp(a, b)
```

Comment:
Does a macro provide any advantage over a constexpr function?

Reply:
> Does a macro provide any advantage over a constexpr function?

Possibly?  This is the way streq is usually formulated (and done in other places in the AP code).  Straight macro sub seems reasonable for this particular function to me.